index.html

```diff
@@ -12,12 +12,15 @@
    <header id="header">
        <h1><a href="index.html">Welcome</a></h1>
        <nav>
-         <ul id="nav">
+         <div class="nav">
+         <ul class="navL" >
             <li> <a href="#works">Porfolio</a></li>
             <li> <a href="#about-myself">about</a></li>
             <li> <a href="#contact">contact</a></li>
          </ul>
-       <img src="images/Enabled.png" alt="Enabled">
+         <a class="closebtn" id="closebtn">&times;</a> 
```

Comment:
- [ ] Please reduce the weight of your close(X) button and make sure it matches the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468) design.

### Your design

![image](https://user-images.githubusercontent.com/83803414/175043568-8da801ad-cc1e-4a1f-a4c4-a5996d9536ca.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/175043633-f81fb8e1-25f2-4c2a-8a6c-2471efae9ebe.png)